tox.ini

```diff
@@ -37,6 +37,7 @@ commands = {posargs}
 #Skip PEP257 violation.
 [flake8]
 ignore = D100,D101,D102,D103,D104,D105,D200,D203,D202,D204,D205,D208,D400,D401
+max-line-length = 100
```

Comment:
No problem, I reverted this in https://github.com/StackStorm/st2mistral/pull/30/commits/5dd169633c0b2256a1753789e33414e38363d844 (and made sure everything passed `tox -epep8` again). Following the rules of the upstream project this is a "part" of makes sense to me (though it wreaks havoc on my regex patterns)!